sonar-bungee/src/main/java/xyz/jonesdev/sonar/bungee/fallback/handler/ReplaceInitialHandler.java

```diff
@@ -0,0 +1,18 @@
+package xyz.jonesdev.sonar.bungee.fallback.handler;
```

Comment:
Please use the same license header as the other files in Sonar (read previous comment).